src/ray/protobuf/usage.proto

```diff
@@ -201,4 +201,20 @@ enum TagKey {
 
   // Train V2 Trainer name (e.g. "TorchTrainer")
   TRAIN_TRAINER = 521;
+
+  // LLM Serve
```

Comment:
Can you document what some of these values are going to be? (Similar to further up in this file)?